document-generator-company-report/src/main/java/uk/gov/companieshouse/document/generator/company/report/mapping/mappers/registrationinformation/ApiToRegistrationInformationMapper.java

```diff
@@ -84,7 +84,7 @@ protected void setIncorporationDateLabel(CompanyProfileApi companyProfileApi, @M
 
         if (companyProfileApi != null && companyProfileApi.getDateOfCreation() != null) {
 
```

Comment:
Can we keep the spacing consistent with the rest of the file? (no white space after the if statement)

Reply:
Whitespace issue resolved